README.md

```diff
@@ -118,7 +118,7 @@ The .Net client comes with implementations for the built-in activation strategie
 - ApplicationHostnameStrategy
 - FlexibleRolloutStrategy
 
-Read more about the strategies in [activation-strategy.md](https://github.com/Unleash/unleash/blob/master/docs/activation-strategies.md).
+Read more about the strategies in [activation-strategy.md](https://github.com/Unleash/unleash/blob/main/docs/activation-strategies.md).
```

Comment:
```suggestion
Read more about the strategies in [activation-strategy.md](https://docs.getunleash.io/user_guide/activation_strategy).
```
Linking to the compiled version of the docs seems more logical, also makes branch name uninteresting :)

Reply:
Particularly since the docs folder no longer contains that markdown file?